### PR TITLE
feat: implement signalingInfo JSON-RPC method

### PR DESCRIPTION
**Pull request by tmpolaczyk**
Example usage:

```
echo '{"jsonrpc":"2.0","id":"1","method":"signalingInfo","params":null}' | cargo run -- -c witnet_01.toml node raw
```

Example output:

```json
{
  "jsonrpc": "2.0",
  "result": {
    "active_upgrades": {
      "WIP0008": 192000,
      "WIP0009-0011-0012": 376320
    },
    "epoch": 501453,
    "pending_upgrades": [
      {
        "bit": 14,
        "end": 4294967295,
        "init": 1096423,
        "period": 100,
        "votes": 0,
        "wip": "WIPXXX1"
      }
    ]
  },
  "id": "1"
}
```

### CODE REVIEW

**Comment by lrubiorod** [comment]
and CLI method?

**Comment by tmpolaczyk** [comment]
> and CLI method?

In another issue

**Comment by lrubiorod** [comment]
> > and CLI method?
> 
> In another issue

😢

File: node/src/actors/messages.rs [hunk 1]
```diff
@@ -589,6 +590,24 @@ impl Message for GetItemSuperblock {
...
+/// Get TAPI Signaling Info
+pub struct GetSignalingInfo {}
+
+/// Result of GetSignalingInfo
+#[derive(Deserialize, Serialize)]
+pub struct SignalingInfo {
+    /// List of WIPs that are already active, and the activation epoch
+    pub active_wips: HashMap<String, Epoch>,
+    /// List of WIPs that are currently being voted
+    pub voting: Vec<BitVotesCounter>,
```
Comment (aesedepece) [inline node/src/actors/messages.rs:602]:
```suggestion
    /// List of protocol upgrades that are already active, and their activation epoch
    pub active_upgrades: HashMap<String, Epoch>,
    /// List of protocol upgrades that are currently being polled for activation signaling
    pub pending_upgrades: Vec<BitVotesCounter>,
```

**Review by aesedepece — CHANGES_REQUESTED** [review]

**Review by aesedepece — APPROVED** [review]

**Pull request merged**